app/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ target_sources(app PRIVATE src/sensors.c)
 target_sources_ifdef(CONFIG_ZMK_DISPLAY app PRIVATE src/display.c)
 target_sources(app PRIVATE src/event_manager.c)
 target_sources_ifdef(CONFIG_ZMK_BLE app PRIVATE src/ble_unpair_combo.c)
+target_sources_ifdef(CONFIG_ZMK_BLE app PRIVATE src/ext_power_generic.c)
```

Comment:
Can I add a Kconfig to control this driver ?

Reply:
Yes, I would say so. Definitely shouldn't be tied to the `ZMK_BLE` Kconfig setting.